src/ferc_xbrl_extractor/arelle_interface.py

```diff
@@ -9,44 +10,41 @@
 from pydantic import BaseModel
 
 
-def load_xbrl(path: str | FileSource.FileSource):
-    """Load XBRL (either taxonomy or individual filing).
-
-    Args:
-        path: URL or local path pointing to an XBRL taxonomy or instance.
-    """
+def _taxonomy_view(taxonomy_source: str | FileSource.FileSource):
+    """Actually use Arelle to get a taxonomy its relationships."""
     cntlr = Cntlr.Cntlr()
     cntlr.startLogging(logFileName="logToPrint")
     model_manager = ModelManager.initialize(cntlr)
-    return ModelXbrl.load(model_manager, path)
+    taxonomy = ModelXbrl.load(model_manager, taxonomy_source)
+
+    view = ViewRelationshipSet(taxonomy, "taxonomy.json", "roles", None, None, None)
+    view.view(XbrlConst.parentChild, None, None, None)
+
+    return taxonomy, view
 
 
-def load_taxonomy(path: str | FileSource.FileSource):
+def load_taxonomy(path: Path):
```

Comment:
I wanted to use `Paths` in as much of our code as possible so I pushed the `str` conversion down to "right before we interface with Arelle."